packages/fields/src/actions/template/index.ts

```diff
@@ -0,0 +1 @@
+export { default as resetTemplateAction } from './reset-template';
```

Comment:
This one was also resetPost because it applied to both "template" and "template parts". It's not out of the question that it will apply to "patterns" later. So I'm not sure if having a "template" folder here makes sense, it might be confusing.

Reply:
It's also unclear to me what's the difference between "base-post" and "common" folders.

Reply:
`base-post` should contain all the actions that belong to this type:

https://github.com/WordPress/gutenberg/blob/cd9fe2f811f0e62bff98e05a0574296f00aeb630/packages/editor/src/dataviews/types.ts#L26-L38


`commont` should contain all the actions that belong to the union type:

https://github.com/WordPress/gutenberg/blob/cd9fe2f811f0e62bff98e05a0574296f00aeb630/packages/editor/src/dataviews/types.ts#L65-L66


It looks like that folder approach is creating more confusion than benefit, I'm going to remove them and create a flat structure 👍 . We can revisit it later!

Reply:
Addressed the feedback with 5a4fb59